tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "@stoplight/scripts/tsconfig.json",
+
+  // target all ts files
   "include": ["."],
-  "exclude": ["**/__*__/**"],
+
   "compilerOptions": {
-    "outDir": "dist",
-    "module": "commonjs"  
```

Comment:
Is this a breaking change?

Reply:
I think it is, because the default for `@stoplight/scripts` is `"module": "esnext",`

Reply:
Done, reverted